pubnub-gson/src/main/java/com/pubnub/internal/PubNubImpl.java

```diff
@@ -144,7 +144,7 @@ public class PubNubImpl extends BasePubNubImpl<
     private final @NotNull BasePNConfiguration configuration;
 
     public PubNubImpl(@NotNull BasePNConfiguration configuration) {
-        super(configuration);
+        super(configuration, "PubNub-Java");
```

Comment:
We could introduce a constant here. What do you think?

Reply:
will do